js/cards.js

```diff
@@ -96,13 +107,17 @@ export async function defineCardComponent() {
       DETAILS_DIALOG_A11Y.show();
     }
 
-    showSubtitle() {
-      this.front.insertAdjacentHTML(
+    getSubtitleText() {
+      return `${this.data["Collector Number"].padStart(3, "0")} ${
+        this.data["Card Display Name"]
+      } (${this.data["Rarity Folder"]})`;
+    }
+
+    addCardSubtitle() {
+      // aria-hidden since the card has its own aria-label handling
+      this.insertAdjacentHTML(
```

Comment:
linking the subtitle to the image with aria-labelledby is usually best practice, but our card html structure is unconventional compared to generic websites. Manipulating aria-label was the quickest way I could achieve conditional labelling

Reply:
moved card-subtitle to card.html and commented this in the file